todoist_api_python/api.py

```diff
@@ -473,6 +473,39 @@ def uncomplete_task(self, task_id: str) -> bool:
         endpoint = get_api_url(f"{TASKS_PATH}/{task_id}/reopen")
         return post(self._session, endpoint, self._token)
 
+    def move_task(
+        self,
+        task_id: str,
+        project_id: str | None = None,
+        section_id: str | None = None,
+        parent_id: str | None = None,
+    ) -> bool:
+        """
+        Move a task.
+
+        Move a task to a different project, section, or parent task.
+        Project_id takes precedence.
+        Moving a task to a section or parent will update its project to match
+        the project of the section or parent task.
+
+        :param task_id: The ID of the task to reopen.
+        :param project_id: The ID of the project to add the task to.
+        :param section_id: The ID of the section to add the task to.
+        :param parent_id: The ID of the parent task.
+        :return: True if the task was moved successfully,
+                 False otherwise (possibly raise `HTTPError` instead).
+        :raises requests.exceptions.HTTPError: If the API request fails.
```

Comment:
```suggestion
        :raises requests.exceptions.HTTPError: If the API request fails.
        :raises ValueError: If neither `project_id`, `section_id`, nor `parent_id` is provided.
```

We should also implement that check. [Here's an example.](https://github.com/radiant-tangent/todoist-api-python/blob/bb34d44477c3cda3738efee41c18664f1c04e9c6/todoist_api_python/api.py#L980-L981)